### PR TITLE
OSL Template Formatting

### PR DESCRIPTION
**Pull request by matthewrsj**

### CODE REVIEW

**Comment by matthewrsj** [comment]
Current OSL Template: http://i.imgur.com/uqOaVFJ.png

File: default-osl.php [hunk 1]
```diff
@@ -229,10 +241,23 @@
...
 
-            <div class="seperator border-bottom-l"></div>
             <?php if ($invoice->invoice_terms) { ?>
                 <h4><?php echo lang('notes'); ?></h4>
                 <p><?php echo nl2br($invoice->invoice_terms); ?></p>
             <?php } ?>
+            <div class="seperator border-bottom-l"></div>
+            <p id="custom-footer">
+              Questions? Please contact Lance Albertson: 541-737-9975
+                <a href="mailto:lance@osl.org">lance@osl.org</a>
```
Comment (MaraJade) [inline default-osl.php]: Lance's email is @osuosl.org

Comment (ramereth) [inline default-osl.php]: It shouldn't have me as the primary email address. For now use accounting@osuosl.org but we may change that too.

Comment (matthewrsj) [inline default-osl.php]: I was going off the template @ramereth provided: http://i.imgur.com/TtAiOYZ.png

**Comment by ramereth** [comment]
@matthewrsj 

[ ] Remove OSU from "OSU Open Source Lab" ... kind of redundant
[ ] Can you make the CASS logo a little bigger so its legible? 
[ ] Make the INVOICE text bold and larger

**Comment by ramereth** [comment]
cass-logo

**Comment by matthewrsj** [comment]
Recommended changes made

**Comment by matthewrsj** [comment]
Made recommended changes:

**Comment by matthewrsj** [comment]
Fixes #8

**Comment by matthewrsj** [comment]
Fixes #8

**Comment by ramereth** [comment]
@matthewrsj can you please add the client email under the "Bill to" section? I'm assuming you can pull that using a variable.

**Comment by matthewrsj** [comment]
Fixes #8

**Comment by matthewrsj** [comment]
Fixes #8

**Comment by matthewrsj** [comment]
Fixes #8

**Comment by ramereth** [comment]
@matthewrsj Can you please add a custom field called "Index Code" and have it shown above the Invoice Date. You can use "ESEP06" as an example index code to show as an example.

**Comment by matthewrsj** [comment]
Fixes #8

**Comment by ramereth** [comment]
@matthewrsj Let's move it below "Amount Due" and change the text to be "Index to Charge"

**Comment by matthewrsj** [comment]
Fixes #8

**Comment by ramereth** [comment]
:+1:

**Comment by matthewrsj** [comment]
Fixes #8

**Pull request merged**